developing/how-to-update-openstack-upgrade-path.md

```diff
@@ -46,6 +46,57 @@ the debian package `bind9`, and we can use `rmadison bind9` to find all the
 versions for charm `designate-bind`. Similarly, for charm `gnocchi`, the
 versions can be found using `cmadison gnocchi`.
 
+The list of package need to be updated are:
+
+```shell
+# Install rmadison
+apt install devscripts
+# Install cmadison
+snap install cmadison
+
+# Find the upper version for the specific releases
```

Comment:
Maybe note that this is specific to these three apps - I thought these were simply three examples (to use for all apps), until I read through the rest of the section.